web/next.config.ts

```diff
@@ -93,23 +96,34 @@ const nextConfig: NextConfig = {
     config.experiments.topLevelAwait = true
     return config
   },
+  sassOptions: {
+    includePaths: ['node_modules'], // the correct option should be loadPaths but somehow that does not work (yet)
+    // TODO: remove this silencing once bootstrap has applied SASS API changes (https://github.com/hodcroftlab/covariants/issues/402)
+    // There are a lot of warnings coming from SASS API changes that bootstrap has not implemented, silencing those
+    // However, this might mask warnings from our own code, so be sure to check from time to time. `quietDeps` seems to
+    // not work completely as intended.
+    quietDeps: true,
+    silenceDeprecations: ['mixed-decls', 'color-functions', 'legacy-js-api', 'global-builtin', 'import'],
+  },
+  output: 'export', // TODO: with the cli command, two threads were used here, no idea how to do it with this config option.
```

Comment:
What about this TODO? Is there an issue that we can reference? Or can we simply delete the comment?